<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CalculatorDialog</class>
 <widget class="QDialog" name="CalculatorDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>555</width>
    <height>452</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Property collection:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lblPropColName">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Properties:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QListWidget" name="lstProperties">
       <property name="viewMode">
        <enum>QListView::ListMode</enum>
       </property>
       <property name="modelColumn">
        <number>0</number>
       </property>
       <property name="uniformItemSizes">
        <bool>false</bool>
       </property>
       <property name="batchSize">
        <number>100</number>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
       <item>
        <property name="text">
         <string>Porosity</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Permeability</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Accoustic Impedance</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Elastic Impedance</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Velocity</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Rhob</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Rhow</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Gamma Ray</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Oil Saturation</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Water Saturation</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Poisson Ratio</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Bulk Module</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Shale</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPlainTextEdit" name="txtScript">
     <property name="font">
      <font>
       <family>Courier New</family>
      </font>
     </property>
     <property name="plainText">
      <string>This is a test.</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="btnRun">
       <property name="toolTip">
        <string>Run script</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="calcresources.qrc">
         <normaloff>:/calcicons/calcplay</normaloff>:/calcicons/calcplay</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnSave">
       <property name="toolTip">
        <string>Save script</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="calcresources.qrc">
         <normaloff>:/calcicons/calcsave</normaloff>:/calcicons/calcsave</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnExprTkHelpPage">
       <property name="text">
        <string>ExprTk Syntax Page</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnDismiss">
       <property name="text">
        <string>Dismiss</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="calcresources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>btnDismiss</sender>
   <signal>clicked()</signal>
   <receiver>CalculatorDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>544</x>
     <y>440</y>
    </hint>
    <hint type="destinationlabel">
     <x>438</x>
     <y>347</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnExprTkHelpPage</sender>
   <signal>clicked()</signal>
   <receiver>CalculatorDialog</receiver>
   <slot>onSyntaxPage()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>127</x>
     <y>428</y>
    </hint>
    <hint type="destinationlabel">
     <x>148</x>
     <y>413</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onSyntaxPage()</slot>
 </slots>
</ui>
